### ISSUE TITLE
Rstudio disconnects when a terminal is open and one installs a package

### ISSUE THREAD

**Issue by stla** [rstudio/rstudio#13002]
Hello,

When a terminal is opened in RStudio and we click on the "Install" button to install a package, then RStudio disconnects:

![Peek 17-04-2023 13-21](https://user-images.githubusercontent.com/4466543/232470702-d280bd46-c4af-47e3-8a85-1ffde55d4913.gif)

### System details

    RStudio Edition : Desktop 
    RStudio Version : 2023.03.0
    OS Version      : Linux Mint 20
    R Version       : 4.2.3

- [x ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
I repro'd on macOS as follows:

* installed 2023.05.0-daily+254
* inside RStudio, create a new R Package project
* click on Terminal tab so it starts a terminal, and keep it visible
* click "Install" in the Build pane

Same result as reported by OP.

**Comment by stla** [comment]
> I also reproduced this on 2022.07, so not Electron-specific.

Yes, I'm facing this issue from a while. Thanks for looking at this.

**Comment by stla** [comment]
No problem if I close the terminal.
